centralserver/i18n/management/commands/update_pot.py

```diff
@@ -1,166 +0,0 @@
-"""
```

Comment:
Now handled in the distributed server's `create_dummy_language_pack` command, which is getting deleted next release anyway to avoid cluttering our codebase.